emva1288/process/data.py

```diff
@@ -275,24 +277,46 @@ def _get_spatial_data(self, d, postfix=''):
         are:
                - *'sum'*: the sum image preserved from input,
                - *'L'*: the number of image summed,
+               - *'M'*: the number of columns for one image
+               - *'N'*: the number of rows for one image
```

Comment:
The standard specifies that M x N images have "M rows and N columns."

Reply:
will fix it shortly.